runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsCryptKCInfobox.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2018, Seth <http://github.com/sethtroll>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.barrows;
+
+import java.awt.Color;
+import java.awt.image.BufferedImage;
+import java.util.Map;
+import javax.inject.Inject;
+import net.runelite.api.Client;
+import net.runelite.client.ui.overlay.infobox.InfoBox;
+import net.runelite.client.ui.overlay.infobox.InfoBoxPriority;
+
+public class BarrowsCryptKCInfobox extends InfoBox
+{
+	private final Client client;
+	private final BarrowsConfig config;
+	private final BarrowsPlugin plugin;
+	private static final int CRYPT_REGION_ID = 14231;
+	private static final int CRYPT_PLANE_ID = 0;
+	private static final String totalKillField = "TOTAL";
+
+	@Inject
+	BarrowsCryptKCInfobox(BufferedImage image, BarrowsPlugin plugin, Client client, BarrowsConfig config)
+	{
+		super(image, plugin);
+		this.plugin = plugin;
+		this.config = config;
+		this.client = client;
+		setTooltip(initializeTooltip());
+		setPriority(InfoBoxPriority.HIGH);
+	}
+
+	@Override
+	public String getText()
+	{
+		Map<String, Integer> monstersKilled = plugin.getMonstersKilled();
+		return Integer.toString(monstersKilled.get(totalKillField));
```

Comment:
Rather than maintain a map entry for total kills, this would be better implemented as either (a) a separate field, or more preferably (b) returning the sum of the map's values, eg.

```java
return Integer.toString(plugin.getMonstersKilled().values().stream().mapToInt(Integer::intValue).sum());
```